pom.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-starter-parent</artifactId>
+        <version>2.5.6</version>
+        <relativePath/> <!-- lookup parent from repository -->
+    </parent>
+
+    <groupId>com.epam.esm</groupId>
+    <artifactId>main</artifactId>
+    <version>1.0.0</version>
+
+    <name>REST-API-Basics</name>
+    <description>REST-API-Basics</description>
+    <packaging>pom</packaging>
+    <properties>
+        <java.version>17</java.version>
```

Comment:
java version is 17? please use version 8, unless you have any arguments to use another one

Reply:
Changed to 8